presentation-api/index.html

```diff
@@ -97,6 +104,15 @@
   }
 });
 
+document.querySelector('#send').addEventListener('click', () => {
+  if (presentationConnection && presentationConnection.state == 'connected') {
+    const message = document.querySelector('#message').value.trim();
+    const lang = document.body.lang || 'en-US';
+    ChromeSamples.log('Sending ' + message + ' to ' + presentationConnection.id);
+    presentationConnection.send(JSON.stringify({string: message, lang: lang}));
```

Comment:
If we were to rename `string` key to `message` in the receiver page, we could do something like

```js
presentationConnection.send(JSON.stringify({message, lang}));
```

It is just a suggestion for readability.